### ISSUE TITLE
Change IRI of based_on

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/COB#148]
`based on` is using a non-standard IRI: http://purl.obolibrary.org/obo/COB_based_on

it would be good if this was either

http://purl.obolibrary.org/obo/COB_0000000

or 

http://purl.obolibrary.org/obo/cob#based_on

**Comment by cthoyt** [comment]
Agreed, thank you. The proliferation of these top-obo properties is a huge pain. I would prefer to see COB_0000000

**Comment by jamesaoverton** [comment]
Since #288 I think that we always use proper numeric terms IDs.
